<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QMainWindow" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1888</width>
    <height>917</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="5" column="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="9">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="Phonon::VideoPlayer" name="videoPlayer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>390</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>600</width>
            <height>390</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <widget class="QPushButton" name="file_browser_button">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>80</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Choose a file</string>
           </property>
          </widget>
          <widget class="QTextBrowser" name="current_file_display">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>221</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QTextBrowser" name="date_created">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>20</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QTextBrowser" name="data_file_display">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>80</y>
             <width>221</width>
             <height>31</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>240</x>
             <y>0</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Date created:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>101</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Current video file:</string>
           </property>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>120</y>
             <width>361</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="1">
             <widget class="QPushButton" name="fast_backward">
              <property name="text">
               <string>&lt;&lt;</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" alignment="Qt::AlignLeft">
             <widget class="Phonon::VolumeSlider" name="volumeSlider"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="start_video">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Start/Pause</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="fast_forward">
              <property name="text">
               <string>&gt;&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QTextBrowser" name="start_point">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>190</y>
             <width>181</width>
             <height>51</height>
            </rect>
           </property>
          </widget>
          <widget class="QTextBrowser" name="end_point">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>260</y>
             <width>181</width>
             <height>51</height>
            </rect>
           </property>
          </widget>
          <widget class="QRadioButton" name="begin_point_rb">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Begin point:</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="end_point_rb">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>240</y>
             <width>101</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>End point:</string>
           </property>
          </widget>
          <widget class="QComboBox" name="tags">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>200</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>skret w prawo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>skret w lewo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>zmiana pasa na lewy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>zmiana pasa na prawy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>wyprzedzanie</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>zatrzymanie na swiat≈Çach</string>
            </property>
           </item>
          </widget>
          <widget class="QPushButton" name="tag_data">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>240</y>
             <width>121</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>Tag data</string>
           </property>
          </widget>
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>Current data file:</string>
           </property>
          </widget>
          <widget class="QLCDNumber" name="lcd_video_time">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>340</y>
             <width>141</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>5</pointsize>
            </font>
           </property>
           <property name="digitCount">
            <number>9</number>
           </property>
          </widget>
          <widget class="QLCDNumber" name="lcd_data_time">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>340</y>
             <width>131</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>5</pointsize>
            </font>
           </property>
           <property name="digitCount">
            <number>9</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>320</y>
             <width>71</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Video time:</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>160</x>
             <y>320</y>
             <width>91</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Plotted data time:</string>
           </property>
          </widget>
          <widget class="QListWidget" name="listWidget">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>20</y>
             <width>321</width>
             <height>361</height>
            </rect>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox">
           <property name="geometry">
            <rect>
             <x>230</x>
             <y>290</y>
             <width>81</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>show speed</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="Phonon::SeekSlider" name="seekSlider">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <property name="iconVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1888</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
